mysql-test/include/filter_file.inc

```diff
@@ -114,7 +123,7 @@ perl;
   {
     chomp;
     s/\015?\012?$//;
-    if (!%column_names)
+    if (!%column_names && $skip_column_names == 0)
```

Comment:
@laurynas-biveinis, we are waiting for your decision about if blueprint is required.